dhis-2/dhis-test-integration/src/test/resources/tracker/event_and_enrollment.json

```diff
@@ -415,7 +415,6 @@
           "value": "value00001",
           "createdAt": "2021-07-01T12:05:00",
```

Comment:
because of this https://github.com/dhis2/dhis2-core/blob/b83162b129c933b7cb97c94775c856d15d611b18/dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/converter/EventTrackerConverterService.java#L255

Reply:
Why do we allow setting the data values created date? Is that not inconsistent 🤔

Reply:
maybe because it's not checking whether this is new data value or updating an existing one